onlineapi/src/main/kotlin/com/codingblocks/onlineapi/models/APIModel.kt

```diff
@@ -102,6 +102,32 @@ data class Course(
 
     }
 
+    @Type("hbcontents")
+    data class CoeChallenge(
+        val name: String,
+        @Relationship("problem")
+        val content: included?=null
```

Comment:
The class name should be Problem not included and always use capital case for classes